packages/cli/generators/observer/templates/observer-template.ts.ejs

```diff
@@ -16,6 +16,13 @@ export class <%= className %> implements LifeCycleObserver {
   ) {}
   */
 
+  /**
+   * This method will be invoked when the application initializes
```

Comment:
Ditto. It would be great to mention again that this method is called only once per application life time.